galaxy/Dockerfile

```diff
@@ -199,4 +203,4 @@ EXPOSE :9002
 VOLUME ["/export/", "/data/", "/var/lib/docker"]
 
 # Autostart script that is invoked during container start
-CMD ["/usr/bin/startup"]
+CMD ["/usr/bin/startCondor.sh"]
```

Comment:
I guess this should be a `-e magic` switch inside of the original startup script, right? Otherwise all the other magic goes away.